src/node_shell_read.js

```diff
@@ -9,18 +9,17 @@ readBinary = (filename) => {
   filename = isFileURI(filename) ? new URL(filename) : filename;
   var ret = fs.readFileSync(filename);
 #if ASSERTIONS
-  assert(ret.buffer);
+  assert(Buffer.isBuffer(ret));
```

Comment:
I think this is more accurately reflects what this assertion was trying to prove.  i.e. that the returned value is a node `Buffer` object.

Reply:
That sounds right, but I recall some differences between node versions... we do have coverage of multiple ones on CI though, so if it passes we should be ok I guess.

Reply:
I was trying to figure out what this assert could possibly be for... is it to detect older node versions that don't return a buffer?  If so, shouldn't we just do that with the version check?  `Buffer.isBuffer` has been around forever so I certainly think it should be fine to use: https://nodejs.org/api/buffer.html#static-method-bufferisbufferobj

Reply:
Oh right, yeah, it was whether a Buffer was returned or not. Yeah, this lgtm.